### PR TITLE
Add add and remove function in todo-list

### PR DESCRIPTION
**Pull request by ecekpo**
---Added ----
-modified changes to the todo-add-remove functionality

### CODE REVIEW

File: src/add_remove_todo.js [hunk 1]
```diff
@@ -0,0 +1,71 @@
+const myList = document.querySelector('.todo-list');
+const todoBtn = document.querySelector('.todo-btn');
+const todoInput = document.querySelector('.todo-input');
+
```
Comment (MeqdamAlqudah) [inline src/add_remove_todo.js]: - [ ] Please fix this issue that the user face when he try to add or remove items
image

**Review by MeqdamAlqudah — CHANGES_REQUESTED** [review]
Hi @ecekpo , 

Great work on making the changes requested by a previous reviewer 👏 
You've done well implementing some of the requested changes, but there are still some which aren't addressed yet.


 ## Highlights


 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions

_Every comment with the [OPTIONAL] prefix won't stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better. Some of them were simply missed by the previous reviewer and addressing them will really improve your application._


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

File: src/add_remove_todo.js (hunk 1 above)
Comment (ecekpo) [reply src/add_remove_todo.js]: Hello @MeqdamAlqudah, 
Thank you for your review. Please, from my end, everything is working fine  as I can add and delete. Kindly, let me know what to be done.  Thank you.
image

Comment (MeqdamAlqudah) [reply src/add_remove_todo.js]: @ecekpo  try to request another review if you didn't face the issue

**Review by MeqdamAlqudah — APPROVED** [review]
Hi @ecekpo , 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 



Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**